src/arity.js

```diff
@@ -1,47 +0,0 @@
-var _curry2 = require('./internal/_curry2');
-
-
-/**
- * Wraps a function of any arity (including nullary) in a function that accepts exactly `n`
- * parameters. Unlike `nAry`, which passes only `n` arguments to the wrapped function,
- * functions produced by `arity` will pass all provided arguments to the wrapped function.
- *
- * @func
- * @memberOf R
- * @sig (Number, (* -> *)) -> (* -> *)
- * @category Function
- * @param {Number} n The desired arity of the returned function.
- * @param {Function} fn The function to wrap.
- * @return {Function} A new function wrapping `fn`. The new function is
- *         guaranteed to be of arity `n`.
- * @deprecated since v0.15.0
- * @example
- *
- *      var takesTwoArgs = function(a, b) {
- *        return [a, b];
- *      };
- *      takesTwoArgs.length; //=> 2
- *      takesTwoArgs(1, 2); //=> [1, 2]
- *
- *      var takesOneArg = R.arity(1, takesTwoArgs);
- *      takesOneArg.length; //=> 1
- *      // All arguments are passed through to the wrapped function
- *      takesOneArg(1, 2); //=> [1, 2]
- */
-module.exports = _curry2(function(n, fn) {
-  // jshint unused:vars
-  switch (n) {
-    case 0: return function() {return fn.apply(this, arguments);};
-    case 1: return function(a0) {return fn.apply(this, arguments);};
-    case 2: return function(a0, a1) {return fn.apply(this, arguments);};
-    case 3: return function(a0, a1, a2) {return fn.apply(this, arguments);};
-    case 4: return function(a0, a1, a2, a3) {return fn.apply(this, arguments);};
-    case 5: return function(a0, a1, a2, a3, a4) {return fn.apply(this, arguments);};
-    case 6: return function(a0, a1, a2, a3, a4, a5) {return fn.apply(this, arguments);};
-    case 7: return function(a0, a1, a2, a3, a4, a5, a6) {return fn.apply(this, arguments);};
-    case 8: return function(a0, a1, a2, a3, a4, a5, a6, a7) {return fn.apply(this, arguments);};
-    case 9: return function(a0, a1, a2, a3, a4, a5, a6, a7, a8) {return fn.apply(this, arguments);};
-    case 10: return function(a0, a1, a2, a3, a4, a5, a6, a7, a8, a9) {return fn.apply(this, arguments);};
-    default: throw new Error('First argument to arity must be a non-negative integer no greater than ten');
```

Comment:
i think this one may be back on the public api in the future, but no problem with removing it for now